shopware/docs
# NEXT-12568 - Migrate e2e testing articles

What should be done?

Create a new article on our GitBook instance which explains how to create end-to-end tests with a plugin.
This can be migrated from our previous documentation.

This article should mention:

    The prerequisite, a working plugin (Refer to the plugin base guide)
    The necessary steps to have a functioning test environment (psh e2e:init or smth, installing cypress, ...)
    The necessary structure for it to work
    Code examples to actually test something
    How to execute these tests (and just these!)

Category: Extensions > Plugins > Testing
Are there already guides in the previous documentation for this?

Yes! But please, don't just copy & paste. Make sure it still works and rewrite it to fit our new writing guidelines!
https://docs.shopware.com/en/shopware-platform-dev-en/how-to/end-to-end-tests-in-plugins

For more information, ask me, Patrick Stahl.
